src/Compilers/VisualBasic/Portable/Binding/Binder_Attributes.vb

```diff
@@ -781,6 +781,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                                     End If
                                     Return CreateErrorTypedConstant(node.Type)
                                 Else
+                                    If node.Syntax.Kind = SyntaxKind.PredefinedCastExpression Then
```

Comment:
Oh, no, I see it is dependent on the context being an attribute. Yes, I see why it belongs here.